pom.xml

```diff
@@ -474,6 +493,34 @@
           <linkJavadoc>true</linkJavadoc>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <configuration>
+          <append>true</append>
+        </configuration>
+        <executions>
+          <execution>
+            <id>agent-for-ut</id>
+            <goals>
+              <goal>prepare-agent</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>agent-for-it</id>
+            <goals>
+              <goal>prepare-agent-integration</goal>
+            </goals>
```

Comment:
Not consistent configuration compared to the Platform

Reply:
When comparing to the platform, here I added both [`jacoco:prepare-agent`](http://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html) and [`jacoco:prepare-agent-integration`](http://www.eclemma.org/jacoco/trunk/doc/prepare-agent-integration-mojo.html), which allow to scan both unit tests and integration tests. Wouldn't it be better?